service-workers/service-worker/global-serviceworker.https.any.js

```diff
@@ -0,0 +1,53 @@
+// META: title=serviceWorker on service worker global
+// META: global=!default,serviceworker
```

Comment:
I see there are some tests today with names like "can-make-payment-event-constructor.https.serviceworker.js". Does it work to make this a "serviceworker.js" test, then you don't need this META?

Reply:
I agree this would be better, but it doesn't seem to work.

https://web-platform-tests.org/writing-tests/testharness.html#auto-generated-test-boilerplate suggests only `window` and `worker` work in the URL.

The `global=!default,serviceworker` line is also directly from their documentation.

Maybe I'm holding it wrong though. I agree it's a little weird having an 'any' test that's limited to just service workers.

Reply:
Ah ok.